apimanagement - 2022-08-01 - track1 sdk

solankisamir opened the pull request:
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

solankisamir committed:
build for track1

solankisamir committed:
update metadata for track1

ArthurMa1978 reviewed (changes requested):
Please fix the tests

solankisamir committed:
remove deprecated operations

KacieKK committed:
record test

mapi test recordings

ShivShankar101 committed:
Wi-17542634 GraphQL API Resolver Tests (#2)

KacieKK committed:
tests (#4)

solankisamir committed:
recordings (#6)

LFZ96 committed:
add authorizationprovider tests (#5)

* add authorizationprovider tests

Add tests for API Management Authorizations feature.

* consolidate tests into one test

* add initial json for authorizationprovidertests

* add loginlinks to test

* Update CreateListUpdateDelete.json

add centraluseuap region details for tests

* Delete CreateListUpdateDelete.json

* add session recordings for authorizationprovidertests

solankisamir committed:
VNET integration test fixes and recordings (#7)

* recordings

* vnet integration tests

malincrist committed:
Added tests for DocumentationOperations (#8)

malincrist committed:
Added tests for auth servers and open id (#9)

solankisamir committed:
new RP recordings

solankisamir committed:
backup tests and recordings

solankisamir committed:
fix test recordings

solankisamir committed:
fix recordings

solankisamir committed:
recordings for failing test

solankisamir committed:
conditional checks in test

solankisamir committed:
cred scan fixes

solankisamir commented:
/azp run

solankisamir commented:
@ArthurMa1978 I am stuck with the failure in the pipeline. Can you help?

ArthurMa1978 committed:
Delete apimanagement_resource-manager.txt
